js/src/forum/components/SignUpModal.js

```diff
@@ -73,8 +73,8 @@ export default class SignUpModal extends Modal {
           type="text"
           placeholder={extractText(app.translator.trans('core.forum.sign_up.username_placeholder'))}
           value={this.username()}
-          onchange={m.withAttr('value', this.username)}
-          disabled={this.loading || this.isProvided('username')}
+          bidi={this.username}
+          disabled={this.loading || this.isProvided('username', this.attrs)}
```

Comment:
That second parameter (`this.attrs`) is not necessary and has not been added to the method (same a few lines below).